From 0730ab085289f647986f1b99d60defcddc74f85e Mon Sep 17 00:00:00 2001
From: ahs <amrita.h.s@intel.com>
Date: Mon, 19 Oct 2020 11:01:16 +0530
Subject: [PATCH] Revert " Enabled AVX2 and FMA compiler Flags for libart
 module."

This reverts commit bf7ce2508c6ed8beb2431fba43af3cdd68a8eb63.
---
 build/Android.bp | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/build/Android.bp b/build/Android.bp
index 946e5a6..fca6351 100644
--- a/build/Android.bp
+++ b/build/Android.bp
@@ -115,25 +115,6 @@ art_global_defaults {
         "-D_LIBCPP_ENABLE_THREAD_SAFETY_ANNOTATIONS",
     ],
 
-    arch: {
-        x86: {
-            avx2: {
-                cflags: [
-                    "-mavx2",
-                    "-mfma",
-                ],
-            },
-        },
-        x86_64: {
-            avx2: {
-                cflags: [
-                    "-mavx2",
-                    "-mfma",
-                ],
-            },
-        },
-    },
-
     target: {
         android: {
             cflags: [
-- 
2.7.4

